shared/attributes.asciidoc

```diff
@@ -108,6 +108,7 @@ Elastic Cloud
 :ess-console: https://cloud.elastic.co?baymax=docs-body&elektra=docs
 :ess-baymax:  ?baymax=docs-body&elektra=docs
 :ece-ref:     https://www.elastic.co/guide/en/cloud-enterprise/current
+:eck-ref:     https://www.elastic.co/guide/en/cloud-on-k8s/current/
```

Comment:
Should we remove trailing `/` for consistency with `ece-ref` above?

Reply:
good catch. I goofed.